### ISSUE TITLE
Had to edit query to get results

### ISSUE THREAD

**Issue by CIRDOC** [Harvard-ATG/omeka-plugin-Elasticsearch#3]
Hi,

I'm trying to use your excellent plugin but had to comment a line in your script to get it working correctly
In the file librairies>Elasticsearch>Helper>index.php
Line 279 :  
`$must_query` = [
                'query_string' => [
                    'query' => $terms,
                  /*  'default_field' => '_all',*/
                    'default_operator' => 'OR'
                ]
            ];
`

Using Elastic 6.2, it seems that default_field_all is deprecated
https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-all-field.html

Anyway, thanks a lot for your job !!!

BenY

**Comment by arthurian** [comment]
Hi @CIRDOC, thanks for the heads up about the `_all` field being deprecated in version 6.2. Much appreciated.

I've created a PR that implements your suggested fix, but I want to do a little more testing before merging it to master. I noticed that the search results aren't *exactly* the same as version 5.x when it comes to nested documents, so I need to do some more investigation to figure out why that is the case.
